testnet/local/README.md

```diff
@@ -0,0 +1,87 @@
+# Docker
```

Comment:
Shall we update the path of all the repositories?

Reply:
Since this README concerns testing the migrated codes, it would be better to have the steps corrected in this PR. Other READMEs need update as well, but they serve for explanatory purposes, so maybe they can be left for future PRs.